daemon/cmd/daemon_main.go

```diff
@@ -1230,6 +1230,8 @@ func initEnv(cmd *cobra.Command) {
 		}
 	case datapathOption.DatapathModeLBOnly:
 		log.Info("Running in LB-only mode")
+		option.Config.LoadBalancerPMTUDiscovery =
```

Comment:
Maybe makes sense to expose it as a flag, so that users could enable/disable it when running in non-LB mode?

Reply:
I originally had that, but for now I've restricted it to XDP only + LB mode. I'm following up to open this up more broadly for skb context, but shouldn't block here, imho.